Tribler/Core/TorrentChecker/torrent_checker.py

```diff
@@ -342,7 +343,8 @@ def _create_session_for_request(self, infohash, tracker_url):
                 request_handled = True
                 break
 
-            if session.can_add_request():
+            #TODO(ardhi) : quickfix for etree.org can't handle multiple infohash in single call
+            if session.can_add_request() and "etree" not in session.tracker_url:
```

Comment:
This looks like a weak check and it should also be moved to `can_add_request()` in any case